css/components/input.css

```diff
@@ -0,0 +1,38 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@400;500;600;700;800&display=swap");
+@import url("../utility.css");
+
+.input {
+    background-color: inherit;
+    border: none;
+    padding: 0.625rem;
+    padding-bottom: 3px;
+    margin: 0.625rem;
```

Comment:
you can set a global variable for spacing as it seems to repeat a few times

Reply:
Sure I will.

Reply:
I've replaced the repeated values with variables

Reply:
Yeah you can resolve this issue now.